src/go/go.mod

```diff
@@ -25,7 +28,9 @@ require (
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/golang/protobuf v1.5.0 // indirect
+	github.com/go-openapi/swag v0.19.15 // indirect
```

Comment:
running `go mod tidy` changed this file somewhat; not sure why there's a mismatch

(also, see later comment about removing swag; which would also change go.mod/go.sum)